src/page/adapter/polynomial-interpolation.js

```diff
@@ -160,14 +160,18 @@
 		var html = '$$f(x)=';
  		for (var i = 0; i < coefficients.length; i++) {
  			var c = coefficients[i];
- 			if (i !== 0) html += '+';
- 			html += roundMathJax(c);
+			if (Math.abs(c) > 1e-30) {
+				if (i !== 0) html += '+ ';
+				html += roundMathJax(c);
 
- 			// different styling for last to segments
- 			if (i == coefficients.length - 2) html += '\\cdot x';
- 			if (i < coefficients.length - 2) html += '\\cdot x^{' + (coefficients.length - i - 1) + '}';
+				// different styling for last to segments
+				if (i == coefficients.length - 2) html += '\\cdot x';
+				if (i < coefficients.length - 2) html += '\\cdot x^{' + (coefficients.length - i - 1) + '}';
+			}
```

Comment:
Catch the case where there are all coefficients 0, e.g.
![image](https://user-images.githubusercontent.com/6556307/78721395-c3918200-7927-11ea-81d7-4a74af1f31f8.png)